index.html

```diff
@@ -37,6 +37,48 @@
     </nav>
 
 
+    <main>
+        <div class="theme">
+            <div class="flex">
+                <div>
+                    <h1 class="h1"> Meow in! </h1>
+                    <p> You know that feeling when you miss sunshine?
+                        Yep, <em> that </em> is how much we missed you. <strong> Welcome back</strong>.
+                    </p>
+                </div>
+                <img src="https://www.smashingmagazine.com/images/smashing-cat/singing-barista.svg" alt="logo"
```

Comment:
Let's add the image locally. This way the website will work, even if smashing magazine changes it's path to the picture.